### ISSUE TITLE
Fix transparency issue in volumetric clouds demo (skiesSample).

### ISSUE THREAD

**Issue by Capewearer** [GaijinEntertainment/DagorEngine#56]
Screenshot shows how barely visible cloud becomes when camera is directed to water.
![изображение](https://github.com/GaijinEntertainment/DagorEngine/assets/145611338/d735576b-e958-47aa-a38f-6099814abc4c)
Hardware: Intel i3-4130 (2 cores 3.4 GHz), AMD R7 260X, 6 Gb RAM. Testing has been done on Windows 7 (may be the problem because it's the unsupported OS).

**Comment by NicSavichev** [comment]
It is a bug in FrameGraph declaration for that sample, will be fixed and uploaded soon.

**Comment by Capewearer** [comment]
Okay, thanks in advance! Could you also explain, what is FrameGraph? Is this a node-based shader editor?

**Comment by NicSavichev** [comment]
No, it is graph for calling render code to customize frame rendering for specific game, more info here
https://gaijinentertainment.github.io/DagorEngine/dagor-render/index/daBFG.html

**Comment by NicSavichev** [comment]
issue fixed in latest release https://github.com/GaijinEntertainment/DagorEngine/releases/tag/dagor_2024_12_04

**Comment by Capewearer** [comment]
> issue fixed in latest release https://github.com/GaijinEntertainment/DagorEngine/releases/tag/dagor_2024_12_04

I can confirm it, even though on other hardware.
